### PR TITLE
Add color_to_hex filter

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: Fluid/Filters/ColorFilters.cs [hunk 1]
```diff
@@ -25,6 +26,32 @@ public static FluidValue ToRgb(FluidValue input, FilterArguments arguments, Temp
             return new StringValue($"rgb({rgbColor.R}, {rgbColor.G}, {rgbColor.B})");
         }
 
+        public static FluidValue ToHex(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            var color = input.ToStringValue();
+            if (!color.StartsWith("rgb(") && color.EndsWith(")") && color.Length >= 10)
```
Comment (sebastienros) [inline Fluid/Filters/ColorFilters.cs:32]: I think the test is wrong

File: Fluid/Filters/ColorFilters.cs [hunk 2]
```diff
@@ -25,6 +26,32 @@ public static FluidValue ToRgb(FluidValue input, FilterArguments arguments, Temp
...
+            {
+                return NilValue.Empty;
+            }
+            
+            var rgbColor = color.Substring(4, color.Length - 5).Split(',');
```
Comment (sebastienros) [inline Fluid/Filters/ColorFilters.cs:37]: nit, you can use the new spread operator. Or do we need to bump the C# version in the project?

Comment (hishamco) [reply Fluid/Filters/ColorFilters.cs:37]: Seems we need to bump the version

File: Fluid/Filters/ColorFilters.cs (hunk 1 above)
Comment (hishamco) [reply Fluid/Filters/ColorFilters.cs:32]: Ya it's 13 ;) I will fix it in `color_to_hsl` PR

File: Fluid/Filters/ColorFilters.cs (hunk 2 above)
Comment (sebastienros) [reply Fluid/Filters/ColorFilters.cs:37]: not worth it then

File: Fluid/Filters/ColorFilters.cs (hunk 1 above)
Comment (sebastienros) [reply Fluid/Filters/ColorFilters.cs:32]: even the && seems fishy

Comment (sebastienros) [reply Fluid/Filters/ColorFilters.cs:32]: or you are missing grouping parens

Comment (hishamco) [reply Fluid/Filters/ColorFilters.cs:32]: No worry, I need to refactor this method in the filter I'm working on, I will keep you updated

Thanks

**Pull request merged**